src/ViewModel/Factory/SiteHeaderFactory.php

```diff
@@ -127,7 +127,7 @@ public function createSiteHeader(Model $item = null, Profile $profile = null) :
             NavLinkedItem::asLink(new Link('About', $this->urlGenerator->generate('about'))),
             NavLinkedItem::asLink(new Link('Community', $this->urlGenerator->generate('community'))),
             NavLinkedItem::asButton(
-                Button::link('Submit my research', $submitUrl ?? $this->submitUrl, Button::SIZE_EXTRA_SMALL)
+                Button::link('Submit my research', $submitUrl ?? $this->submitUrl, Button::SIZE_EXTRA_SMALL, Button::STYLE_DEFAULT, true, false, 'submitResearchButton')
```

Comment:
@giorgiosironi id currently set to `submitResearchButton `. Feel free to change as appropriate.

Reply:
Have no preference as long as it's stable in time.